### PR TITLE
[react] add translate prop to HTMLAttributes

### PR DESCRIPTION
**Pull request by noomorph**
Hello!

Honestly, I was surprised that I am not able to set `translate` attribute on an HTML element in React + TypeScript.

[§ 3.2.6.3 The translate attribute (WHATWG)](https://html.spec.whatwg.org/multipage/dom.html#the-translate-attribute)

Was the omission of this attribute discussed somewhere or somewhen, why is that so? Or that happened accidentally?

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:

- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://html.spec.whatwg.org/multipage/dom.html#the-translate-attribute

### CODE REVIEW

**Comment by noomorph** [comment]
I wonder if the build is failing because of `react-native-modal` misconfiguration... :thinking: 
```
6> Error: Errors in typescript@next for external dependencies:
6> node_modules/react-native-modal/dist/modal.d.ts(1,8): error TS1259: Module '"/home/noomorph/Projects/charity/DefinitelyTyped/types/react/index"' can only be default-imported using the 'esModuleInterop' flag
6> node_modules/react-native-modal/dist/modal.d.ts(3,8): error TS1192: Module '"/home/noomorph/Projects/charity/DefinitelyTyped/types/prop-types/index"' has no default export.
6>
6>     at /home/noomorph/Projects/charity/DefinitelyTyped/node_modules/dtslint/bin/index.js:191:19
6>     at Generator.next (<anonymous>)
6>     at fulfilled (/home/noomorph/Projects/charity/DefinitelyTyped/node_modules/dtslint/bin/index.js:6:58)
```

**Comment by noomorph** [comment]
All items on the checklist have been checked.

File: types/react/test/elementAttributes.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    <span hidden />,
+    <span id="s" />,
+    <span lang="art-x-tokipona" />,
+    <input placeholder="placeholder" />,
+    <span slot="my-text" />,
+    <span spellCheck />,
+    <span tabIndex={0} />,
+    <span title="title" />,
+    <span role="button" />,
+    <span role="button" />,
```
Comment (eps1lon) [inline types/react/test/elementAttributes.tsx]: nit: duplicated

**Review by eps1lon — APPROVED** [review]
Thanks. If it fails on master then it will get fixed at some point. Try rebasing the next time the bot bothers you with failing tests.

File: types/react/test/elementAttributes.tsx (hunk 1 above)
Comment (noomorph) [reply types/react/test/elementAttributes.tsx]: Thanks! Fixed and supplemented with more test cases.

**Comment by noomorph** [comment]
@eps1lon, finally, I think we can merge, the build is passing.

**Comment by eps1lon** [comment]
> @eps1lon, finally, I think we can merge, the build is passing.

I don't have write access. Since all is green a wild collaborator will soon appear. They will use "merge" which is very effective.

**Pull request merged**